osu.Game.Rulesets.Catch/Judgements/CatchJudgement.cs

```diff
@@ -2,11 +2,49 @@
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
 
 using osu.Game.Rulesets.Judgements;
+using osu.Game.Rulesets.Scoring;
 
 namespace osu.Game.Rulesets.Catch.Judgements
 {
     public class CatchJudgement : Judgement
     {
-        // todo: wangs
+        public override HitResult MaxResult => HitResult.Perfect;
+
+        /// <summary>
+        /// The positional hit offset.
+        /// </summary>
+        public float PositionOffset;
```

Comment:
I don't think we'll be needing this - judgement will probably be at the plate itself, which the plate already knows the position for.

![screen shot 2018-06-13 at 1 15 18 pm](https://user-images.githubusercontent.com/1329837/41330002-db2a97da-6f0b-11e8-976c-6ac5d03ec2d5.png)

Reply:
Can you explain this comment? `PositionOffset` would be used to store the position relative to the center of the plate for potentially showing as a statistic on the results screen, no? It should have nothing to do with visuals.

Reply:
Do you have a design for this? This is the first time I've seen this discussed, and I thought we didn't want to implement things that may not even be implemented this way in the future.

Reply:
No, I'm just explaining the point of `PositionalOffset` inside judgements (this is how it's implemented in osu! for instance)

Reply:
( just confirming your comment isn't related to this at all. not sure why there's a screenshot for instance)

Reply:
`PositionalOffset` has no function in osu! currently either, so I would not be against removing it there too. I don't see why this was added here either.

Again, this is the first time I've heard of using this as a statistic on the results screen, I thought it'd be used for the vertical lines that shoot out of the catcher instead.

Regardless the fact that the confusion over this member exists should tell you enough about its existence.

Reply:
Judgements only contain things which will be used for statistics/judgement/scoring purposes, not display purposes. It was originally added to figure out how we would be structuring things going forward.

And yes if are going to use the design-must-exist rule we do have designs for this display.

Reply:
As long as you populate both this and the osu!-side one.